app/server.go

```diff
@@ -223,6 +223,8 @@ func NewServer(options ...Option) (*Server, error) {
 		mlog.Error(err.Error())
 	}
 
+	mlog.Info("Go Version Detected: " + runtime.Version())
+
 	// This is called after initLogging() to avoid a race condition.
 	mlog.Info("Server is initializing...")
```

Comment:
Maybe add the info as a field to this line. It can be thought of as that the server is initializing with certain parameters. And we can just say "Go version".

Reply:
Fixed it. now it says 
```go
Server is initializing with Go version 
```

Reply:
Sorry I meant something like `mlog.Info("Server is initializing...", mlog.String("Go version", runtime.Version()))`. It is better suited for log processing tools.

Reply:
Done